cmd/gcp-controller-manager/app/gke_approver.go

```diff
@@ -595,6 +608,16 @@ func groupHasInstance(opts GCPConfig, zone, groupName string, instanceID uint64)
 	return false, nil
 }
 
+func parseInstanceGroupURL(ig string) (name, location string, err error) {
```

Comment:
Up to you, but a regex may be more readable here. Readers may not know how does the instance group url looks like.